model/src/main/java/org/cloudfoundry/identity/uaa/util/UaaStringUtils.java

```diff
@@ -47,6 +47,7 @@ public final class UaaStringUtils {
     private static final Pattern CTRL_PATTERN = Pattern.compile("[\n\r\t]");
 
     public static final String EMPTY_STRING = "";
+    public static final String[] EMPTY_STRING_ARRAY = {};
```

Comment:
I don't understand the point of these 2 constants. I would rather see `""` and `{}` in the code than that.

Not having [magic values](https://en.wikipedia.org/wiki/Magic_number_(programming)) in the code is great. But replacing a magic `2` in the code by having `public final int TWO = 2` doesn't solve the problem. It just creates a useless redirect that makes the code more difficult to understand.

Reply:
IMHO, using the `EMPTY_STRING` constant instead of `""` makes it more obvious that the empty string is actually intended and was not just forgotten.

Reply:
I also just removed the `EMPTY_STRING_ARRAY` constant.